packages/jasmine/src/jasmine_node_test.bzl

```diff
@@ -61,12 +67,21 @@ def jasmine_node_test(
     all_data += [Label("@bazel_tools//tools/bash/runfiles")]
 
     # If the target specified templated_args, pass it through.
-    templated_args = kwargs.pop("templated_args", []) + ["$(location :%s_devmode_srcs.MF)" % name]
+    templated_args = kwargs.pop("templated_args", [])
+    templated_args.append("$(location :%s_devmode_srcs.MF)" % name)
 
     if coverage:
-        templated_args = templated_args + ["--coverage"]
+        templated_args.append("--coverage")
+    else:
+        templated_args.append("--nocoverage")
+
+    if config_file:
+        # Calculate a label relative to the user's BUILD file
+        pkg = Label("%s//%s:__pkg__" % (native.repository_name(), native.package_name()))
+        all_data.append(pkg.relative(config_file))
+        templated_args.append("$(location %s)" % config_file)
     else:
-        templated_args = templated_args + ["--nocoverage"]
+        templated_args.append("--noconfig")
```

Comment:
Doesn't look like config_file makes it into runfiles unless I'm missing something?

Reply:
it goes into `all_data` which puts it in the `data` attr of the nodejs_test - so yeah it should be